tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/generator/CodeGenVisitor.java

```diff
@@ -736,6 +740,16 @@ public void visit(AbortStmt abortStmt) {
     }
 
     @Override
+    public void visit(NamespaceDeclarationStmt namespaceDeclarationStmt) {
+
```

Comment:
Should we throw NotImplimentedException here ?

Reply:
this is fixed in here >> https://github.com/wso2/product-ei/pull/1216/files#diff-8900249e8d03a837a913b74ecd155c9dL757